nixos/modules/config/users-groups.nix

```diff
@@ -139,6 +139,12 @@ let
         description = "The user's home directory.";
       };
 
+      homeMode = mkOption {
+        type = types.strMatching "[0-7]{1,5}";
+        default = "700";
+        description = "The user's home directory mode in numeric format. See chmod(1).";
```

Comment:
I think it would be important to document that the `homeMode` only has an effect if `createHome` is true, and is only applied when creating the home.

Reply:
Agreed, I've updated the description.